auth-server-open-api-spec.yaml

```diff
@@ -516,6 +518,27 @@ components:
         - $ref: '#/components/schemas/access-incoming'
         - $ref: '#/components/schemas/access-outgoing'
       description: The access associated with the access token is described using objects that each contain multiple dimensions of access.
+    limits-outgoing-recurring:
+      title: limits-outgoing-recurring
+      description: Open Payments specific property that defines the limits under which outgoing payments can be created.
+      type: object
+      properties:
+        receiver:
+          type: string
+          format: uri
+          description: URL for the incoming payment or ILP STREAM Connection.
```

Comment:
This could be made its own schema to avoid repeating.
Ditto for `interval`

Reply:
Deferred to #156